rtb/piano-di-progetto/src/pianificazione.tex

```diff
@@ -1,7 +1,578 @@
-
+\pagebreak
 \section{Pianificazione}
-\subsection{Analisi}
-\subsection{Proof of concept}
-\subsection{Progettazione e codifica}
-\subsection{Valutazione e collaudo}
+\subsection{Verso la RTB}
+\subsubsection{Primo periodo} 
+Inizio: 8/11/2023 \\
+Fine: 24/11/2023 \\
+
+\textbf{Preventivo orario}
+
+\begin{tblr}{
+    colspec={|X[5cm]|X[.5cm]|X[.5cm]|X[.5cm]|X[.5cm]|X[.5cm]|X[.5cm]|X[3.5cm]},
+    row{odd}={bg=white},
+    row{even}={bg=lightgray},
+    row{1}={bg=black,fg=white},
+    row{8}={bg=black,fg=white}
+    }
+    
+    Nominativo & Re & Am & An & Pg & Pr & Vf & Ore Totali \\ \hline
+    Alberto C.    & -  & -  & 5  & -  & -  & 1  & 5 \\ \hline
+    Bibal El M.   & -  & -  & 4  & -  & -  & 1  & 5 \\ \hline
+    Alberto M.    & 3  & 2  & -  & -  & -  & -  & 5 \\ \hline
+    Alex S.       & 2  & 3  & -  & -  & -  & -  & 5 \\ \hline
+    Iulius S.     & -  & -  & 4  & -  & -  & 1  & 5 \\ \hline
+    Giovanni Z.   & -  & -  & 4  & -  & -  & 1  & 5 \\ \hline
+    Totale        & 5  & 5  & 17 & 0  & 0  & 4  & 30 \\ \hline
+    
+\end{tblr}
+
+\textbf{Preventivo economico}
+
+\begin{tblr}{
+colspec={|X[5cm]|X[3.5cm]|X[1.5cm]|X[3.5cm]},
+row{odd}={bg=white},
+row{even}={bg=lightgray},
+row{1}={bg=black,fg=white},
+row{8}={bg=black,fg=white}
+}
+
+Ruolo & Costo orario (€/h) & N. Ore & Costo totale (€)  \\ \hline
+Responsabile      & 30,00 &  5  &   150,00 \\ \hline
+Amministratore    & 20,00 &  5  &   100,00 \\ \hline
+Analista          & 25,00 &  17 &   425,00 \\ \hline
+Progettista       & 25,00 &   0 &     0,00 \\ \hline
+Programmatore     & 15,00 &   0 &     0,00 \\ \hline
+Verificatore      & 15,00 &   4 &    60,00 \\ \hline
+Totale & \SetCell[c=1]{c} & 192 &   735,00 \\ \hline
+
+\end{tblr}
+
+\paragraph{Attività svolte} 
+\subparagraph{}
+Le attività svolte dai membri del gruppo nel primo periodo sono state:
+\begin{itemize}
+    \item Concepimento della prima versione del Way of Working;
+    \item Ricerca bibliografica finalizzata a costruire il body of knowledge necessario alla comprensione della documentazione
+    che dovrà essere prodotta nella fase di RTB;
+    \item Studio dei seguenti documenti:
+    \begin{itemize}
+        \item ISO/IEC 12207 : Processi del ciclo di vita del software;
+        \item ISO/IEC TR 19759: Software Engineering - Guide to the Software Engineering Body of Knowledge; 
+        \item IEEE 830: Pratiche raccomandate per la specifica dei requisiti software;
```

Comment:
Ti direi di aggiungere già i riferimenti a questi standard nella sezione riferimenti in introduzione.tex

Reply:
oki,però li terrei anche qua comunque